### PR TITLE
Fix bug where gas param was not a string

### LINKED ISSUE #2598: uat - Tx Params should always be a string
I'm currently unable to send a tx from a branch off `uat` because it is providing the `gas` parameter as a number instead of a string. (Full disclosure: I've modified this branch a bit, but I haven't modified gas inputs)

screen shot 2017-11-17 at 12 43 16 pm

This behavior is enforced by [the tx-state-manager here](https://github.com/MetaMask/metamask-extension/blob/master/app/scripts/lib/tx-state-manager.js#L95).

### PR DESCRIPTION
**Pull request by danfinlay**
Prevented sending transactions.

Fixes #2598

### CODE REVIEW

File: ui/app/components/customize-gas-modal/index.js [hunk 1]
```diff
@@ -244,7 +244,7 @@ CustomizeGasModal.prototype.render = function () {
           min: 1,
           // max: 100000,
           step: 1,
-          onChange: value => this.convertAndSetGasLimit(value),
+          onChange: value => this.convertAndSetGasLimit(String(value)),
```
Comment (danjm) [inline ui/app/components/customize-gas-modal/index.js]: I don't think this change is needed. The `conversionUtil` call on line 155 will always return a string.

Comment (danjm) [inline ui/app/components/customize-gas-modal/index.js]: I don't think this change is the. The `conversionUtil` call on line 155 will always return a string.

**Comment by danjm** [comment]
This looks good. I left one comment, but it can be ignored if you prefer.

Did you find this bug when reproducing the bug described in https://github.com/MetaMask/metamask-extension/issues/2599?

**Comment by danfinlay** [comment]
Made the requested change. I did find this bug when fixing #2599.

**Pull request merged**